hack/tools/install.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
Move hack/tools/install.sh to hack/install.sh

Reply:
Ok